spec/models/library_entry_spec.rb

```diff
@@ -134,6 +135,49 @@
     end
   end
 
+  describe 'timestamp validation' do
+    let!(:anime) { create(:anime) }
```

Comment:
With `before(:each)` does it keep referencing the same object so i can modify it across tests? If not is there a way to do that without `let`? Or should I not be doing that at all and create new objects for each `it` block?